charts/aws-mountpoint-s3-csi-driver/templates/mountpoints3podattachments-crd.yaml

```diff
@@ -124,8 +124,10 @@ spec:
             - workloadFSGroup
             type: object
         type: object
+    {{- if semverCompare ">=1.32.0-0" .Capabilities.KubeVersion.Version }}
     selectableFields:
     - jsonPath: .spec.nodeName
+    {{- end }}
```

Comment:
For customers that would upgrade K8s from e.g. 1.31 to 1.33, would they be able to utilize `selectableFields` if they would update CSI Driver afterwards? I.e. does K8s allow adding `selectableFields` to existing CRD version `v2`?

Or would they have to re-install the driver fully. 

I am worried that when customers would update CSI Driver, our code in `IsSelectableFieldsSupported()` would start returning `true` and this will cause error in our Node during listing of CRDs using `nodeName` selector. (Because CRD does not mention that `nodeName` is selectable field)